source/FreeRTOS_Sockets.c

```diff
@@ -100,6 +100,15 @@
 #define sock80_PERCENT     80U         /**< 80% of the defined limit. */
 #define sock100_PERCENT    100U        /**< 100% of the defined limit. */
 
+
+#if ( ( ipconfigHAS_DEBUG_PRINTF != 0 ) || ( ipconfigHAS_PRINTF != 0 ) )
+
+/**
+ * @brief A buffer for prvSocketProps to return socket property in string.
+ */
+    static char pucSocketProps[ 92 ];
```

Comment:
Why are we moving this variable out of the `prvSocketProps()` if its only used inside that function?

Reply:
This is used to validate in UT.
If the variable is declared inside the function, we can't extern it in the UT to verify the result.

Reply:
Because we removed keyword `static` as preprocess for unit test. This variable became local variable that we can't access outside of function. Move it to global section to keep the buffer accessible for unit test.

Reply:
This might give coverity error, please check

Reply:
I believe the pointer returned by `prvSocketProps()` is the address of `pucSocketProps` and we could use that to access the buffer during the unit tests, this will involve maybe calling the `prvSocketProps()` twice in a test or we could store the address in a global variable during the initial test and then use it in further tests.

Reply:
Debug print is disabled in coverity scan, so this change doesn't give any coverity error.
But I did find some coverity issues. Just updated for them.
Thanks for notifying.